### PR TITLE
HRIS-208 [Markup] Create Schedule Modal

### PR DESCRIPTION
**Pull request by jsvelte**
## Issue Link

- https://framgiaph.backlog.com/view/HRIS-208

## Definition of Done

- [x] Created `ViewScheduleMembersModal`
- [x] Displayed Member List 
- [x] Implement Simple Client Side Search Functionality
- [x] Improve Tab Page Animation with Framer motion

## Notes

Figma Design
- https://www.figma.com/file/w3lR4Elj1q5yZgK3Xw0eIx/HRIS-Wireframe?node-id=2576-28630&t=jhFnYMvC5XGJg7j5-4

#### Docker
run `docker compose up --build`

#### Local
- run `cd client && npm run dev`
- run `cd api && dotnet run watch`

## Expected Output

- It should now have a markup modal for View Schedule Members
- Simple Client Side Search Functionality of a Members

## Screenshots/Recording
[modal.webm](https://user-images.githubusercontent.com/108642414/234168144-c0b9ca86-ae21-44e0-9f04-10cd05bddc14.webm)

#### Animated Tab
[animated tab.webm](https://user-images.githubusercontent.com/108642414/234168270-07056046-f62e-4961-91b3-202880014a04.webm)

### CODE REVIEW

File: client/src/components/molecules/ViewScheduleMembersModal/index.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      <main>
+        {/* For Search Field */}
+        <section className="group relative px-7 py-4">
+          <div className="absolute inset-y-0 flex items-center pl-3">
+            <Search className="h-5 w-5 text-slate-400/70 group-focus-within:text-amber-500" />
+          </div>
+          <Input
+            type="text"
+            color="warning"
+            placeholder="Fine Members"
```
Comment (arden-dave) [inline client/src/components/molecules/ViewScheduleMembersModal/index.tsx:46]: Is it supposed to be "Find"?

**Review by arden-dave — APPROVED** [review]
Approve. 👍

File: client/src/pages/schedule-management.tsx [hunk 2]
```diff
@@ -143,7 +147,7 @@ const ScheduleManagement: NextPage = (): JSX.Element => {
         {
           onSuccess: () => {
             void queryClient.invalidateQueries().then(() => {
-              toast.success('Updated Employee Schedule Successfully')
+              toast.success('Updated Successfully')
             })
```
Comment (abduljalilpalala) [inline client/src/pages/schedule-management.tsx:151]:
```suggestion
              toast.success('The schedule has been updated successfully!')
            })
```

File: client/src/pages/schedule-management.tsx [hunk 3]
```diff
@@ -115,7 +120,7 @@ const ScheduleManagement: NextPage = (): JSX.Element => {
                 queryKey: ['GET_ALL_EMPLOYEE_SCHEDULE']
               })
               .then(() => {
-                toast.success('Created new Employee Schedule Successfully')
+                toast.success('Added New Schedule Successfully')
                 handleReset()
```
Comment (abduljalilpalala) [inline client/src/pages/schedule-management.tsx:124]:
```suggestion
                toast.success('The schedule has been added successfully!')
                handleReset()
```

**Pull request merged**